### ISSUE TITLE
Microphone Issue

### ISSUE THREAD

**Issue by indraAsLesmana** [cstrouse/Lenovo-M73-Tiny-Hackintosh#1]
Microphone via headset doesn't work.

**Comment by cstrouse** [comment]
Is your machine also a 10AY version? Layout 3 works for me for both input and output with the headset from the Galaxy S9. You may have a slightly different version of the audio controller or different BIOS options.

Can you respond with your BIOS configuration and the full Lenovo model number for your M73?

**Comment by indraAsLesmana** [comment]
Yes its 10AY0003AU? i didn't change any config. its must be layout-id 3 also.

BIOS just change Video Graphic to 32/64mb but not help. tray boot UEFI/Legacy.
but i have difference Processor, me with i5 4570t.

Sound from Display-port work great, but cracking with headset, microphone detected with name title "internal microphone" type "Built-in" but not work.

**Comment by cstrouse** [comment]
Try the other layouts supported by this audio codec in the [AppleALC repo](https://github.com/acidanthera/AppleALC/tree/master/Resources/ALC283). I would start with 44 as the M93 uses it and it may work for you.

You'll need some changes to the `config.plist` to properly support your CPU/iGPU but this shouldn't have anything to do with the audio as far as I can tell.

The only thing I changed after loading defaults in the BIOS settings were the video memory bump to 64MB and I changed it to UEFI-only.
